### PR TITLE
Rename messaging to dmsg

### PR DESCRIPTION
**Pull request by Darkren**
Fixes https://github.com/SkycoinPro/skywire-services/issues/26

 Changes:	
- Renamed messaging to dmsg
- Tested it with the integration env

### CODE REVIEW

File: cmd/dmsg-server/commands/root.go
Comment (nkryuchkov) [inline cmd/dmsg-server/commands/root.go]: @evanlinjin I remember you said that lowercase spelling would be correct (`dmsg`). Is `DMSG` appropriate as well?

Comment (evanlinjin) [reply cmd/dmsg-server/commands/root.go]: @nkryuchkov I've always stuck with all lowercase. To be honest, I'm not sure what is best haha. What convention do you think we should have?

Comment (nkryuchkov) [reply cmd/dmsg-server/commands/root.go]: @evanlinjin @Darkren Maybe `DMsg`?

**Review by evanlinjin — CHANGES_REQUESTED** [review]
As this is a milestone 2 PR, we should use binaries from the `dmsg` repo (`dmsg-server` and `dmsg-discovery`), as they have been moved there.

https://github.com/evanlinjin/dmsg-1/tree/feature/m2-yamux/cmd

File: cmd/dmsg-server/commands/root.go
Comment (evanlinjin) [reply cmd/dmsg-server/commands/root.go]: @nkryuchkov technically correct. But someone needs to go in and change it all haha.

Comment (Darkren) [reply cmd/dmsg-server/commands/root.go]: @evanlinjin @nkryuchkov should I change it? and what for? I didn't get the resume lol

**Comment by Darkren** [comment]
@evanlinjin this makes this PR blocked by this one #115 . In #115  and its bro PR from services repo I use binaries from the `dmsg` repo. if this is the only problem here I think it would be better to merge this one for now. after finishing and merging #115 everything should be in its place

File: cmd/dmsg-server/commands/root.go
Comment (evanlinjin) [reply cmd/dmsg-server/commands/root.go]: @Darkren @nkryuchkov Let's stick with lowercase `dmsg` for now. As that's how it is written everywhere else. We can talk about changes in the future.

Comment (Darkren) [reply cmd/dmsg-server/commands/root.go]: Done this

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**
